src/main/java/com/conveyal/r5/analyst/cluster/PathResult.java

```diff
@@ -38,7 +38,7 @@ public class PathResult {
      * These results are returned to the backend over an HTTP API so we don't want to risk making them too huge.
      * This could be set to a higher number in cases where you know the result return channel can handle the size.
      */
-    public static int maxDestinations = 5000;
+    public static int MAX_PATH_DESTINATIONS = 5_000;
```

Comment:
It looks like this was originally left as a variable to allow adjustments, but it's not clear how that adjustment would be made now given how these objects are created. So it does make sense to treat this as a constant, and all-caps is conventionally reserved for constants. But in that case there's a missing `final` here.

Similar comment for the DATA_COLUMNS below, looks like we're missing a `final`.